Mesh_3/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# This is the CMake script for compiling this folder.
+
+cmake_minimum_required(VERSION 3.1...3.23)
+project(Mesh_3)
+
+if(CGAL_ENABLE_TESTING)
+  enable_testing()
+endif()
+
```

Comment:
Useless, because it will be done in sub-directory, as soon as `CGALConfig.cmake` is found:

`CGALConfig.cmake` includes `CGAL_CreateSingleSourceCGALProgram.cmake`, that includes `CGAL_add_test.cmake`, and the later has:

https://github.com/CGAL/cgal/blob/2c13f1fe6bcb7496b5f5c68095790ae4f894f3dc/Installation/cmake/modules/CGAL_add_test.cmake#L10-L14

So:
```suggestion
```